src/TODOApp.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import TODOAddForm from './TODOAddForm';
+import TODOList from './TODOList';
+import AppHead from './AppHead';
+import './App.css';
+
+var generator = () => {
+  return Math.floor((1 + Math.random()) * 0x10000)
```

Comment:
Move this method to some utility file.
Also don't use var until it is required. Here `const` will work.